app/assets/javascripts/discourse/app/lib/plugin-api.gjs

```diff
@@ -1590,6 +1667,12 @@ class PluginApi {
    * })
    */
   addPostTransformCallback(callback) {
+    // TODO (glimmer-post-stream) what should replace this API?
```

Comment:
This shouldn't be needed at all under Glimmer post-stream, right?

If people want to customize the post model, they can use `modifyClass`

Reply:
I removed the TODO and changed the deprecation message to just state that the method is deprecated.

Should we offer guidance that the user needs to modify the Post model?